internal/ingress/annotations/annotations.go

```diff
@@ -132,48 +134,47 @@ type Extractor struct {
-			"SecureUpstream":       secureupstream.NewParser(cfg),
-			"ServerSnippet":        serversnippet.NewParser(cfg),
-			"ServiceUpstream":      serviceupstream.NewParser(cfg),
-			"SessionAffinity":      sessionaffinity.NewParser(cfg),
-			"SSLPassthrough":       sslpassthrough.NewParser(cfg),
-			"UsePortInRedirects":   portinredirect.NewParser(cfg),
-			"UpstreamHashBy":       upstreamhashby.NewParser(cfg),
-			"LoadBalancing":        loadbalancing.NewParser(cfg),
-			"UpstreamVhost":        upstreamvhost.NewParser(cfg),
-			"Whitelist":            ipwhitelist.NewParser(cfg),
-			"Denylist":             ipdenylist.NewParser(cfg),
-			"XForwardedPrefix":     xforwardedprefix.NewParser(cfg),
-			"SSLCipher":            sslcipher.NewParser(cfg),
-			"Logs":                 log.NewParser(cfg),
-			"InfluxDB":             influxdb.NewParser(cfg),
-			"BackendProtocol":      backendprotocol.NewParser(cfg),
-			"ModSecurity":          modsecurity.NewParser(cfg),
-			"Mirror":               mirror.NewParser(cfg),
-			"StreamSnippet":        streamsnippet.NewParser(cfg),
+			"Aliases":               alias.NewParser(cfg),
+			"BasicDigestAuth":       auth.NewParser(auth.AuthDirectory, cfg),
+			"Canary":                canary.NewParser(cfg),
+			"CertificateAuth":       authtls.NewParser(cfg),
+			"ClientBodyBufferSize":  clientbodybuffersize.NewParser(cfg),
+			"ConfigurationSnippet":  snippet.NewParser(cfg),
+			"Connection":            connection.NewParser(cfg),
+			"CorsConfig":            cors.NewParser(cfg),
+			"CustomHTTPErrors":      customhttperrors.NewParser(cfg),
+			"CustomResponseHeaders": customresponseheaders.NewParser(cfg),
+			"DefaultBackend":        defaultbackend.NewParser(cfg),
+			"FastCGI":               fastcgi.NewParser(cfg),
+			"ExternalAuth":          authreq.NewParser(cfg),
+			"EnableGlobalAuth":      authreqglobal.NewParser(cfg),
+			"HTTP2PushPreload":      http2pushpreload.NewParser(cfg),
+			"Opentracing":           opentracing.NewParser(cfg),
+			"Proxy":                 proxy.NewParser(cfg),
+			"ProxySSL":              proxyssl.NewParser(cfg),
+			"RateLimit":             ratelimit.NewParser(cfg),
+			"GlobalRateLimit":       globalratelimit.NewParser(cfg),
+			"Redirect":              redirect.NewParser(cfg),
+			"Rewrite":               rewrite.NewParser(cfg),
+			"Satisfy":               satisfy.NewParser(cfg),
+			"SecureUpstream":        secureupstream.NewParser(cfg),
+			"ServerSnippet":         serversnippet.NewParser(cfg),
+			"ServiceUpstream":       serviceupstream.NewParser(cfg),
+			"SessionAffinity":       sessionaffinity.NewParser(cfg),
+			"SSLPassthrough":        sslpassthrough.NewParser(cfg),
+			"UsePortInRedirects":    portinredirect.NewParser(cfg),
+			"UpstreamHashBy":        upstreamhashby.NewParser(cfg),
+			"LoadBalancing":         loadbalancing.NewParser(cfg),
+			"UpstreamVhost":         upstreamvhost.NewParser(cfg),
+			"Whitelist":             ipwhitelist.NewParser(cfg),
```

Comment:
Appears tests are failing because DenyList got removed, readding it should at least fix that test. @longwuyuan 
```suggestion
			"Whitelist":             ipwhitelist.NewParser(cfg),
			"Denylist":              ipdenylist.NewParser(cfg),
```

Reply:
@philnichol I saw another commit to this file after yours and I can't see that one removing Denylist. And your commit added it.

My git is not so good. Do you have tips on where this "Denylist" disappeared from main, after your commit to main.

Reply:
@philnichol , sorry, ignore last comment from me. I can see "Denylist" is in main, on line 168.
So @satyamz , are you removing it ? I am checking though but first glance did not see it so my bad.

Reply:
No problem @longwuyuan , it's still in main https://github.com/kubernetes/ingress-nginx/blob/main/internal/ingress/annotations/annotations.go#L168
Looks like it was just removed as part of this branch/PR, maybe just got accidentally removed

Reply:
@philnichol and @longwuyuan  Thanks for the help. That explains it. I think I accidentally removed `Denylist` from branch when I was resolving conflicts in my branch. I added it back and test succeeded. Thanks again!